source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -1,187 +1,95 @@
+- read about [what to do first](/switching_to_live/#before-you-start)
+- [request a live GOV.UK Pay account](/switching_to_live/#1-request-a-live-gov-uk-pay-account)
 
-You should follow these instructions in order.
+You must also be an admin on your GOV.UK Pay account.
 
-You can also read about setting up live
-[ePDQ](/switching_to_live/set_up_a_live_epdq_account) and
-[Smartpay](/switching_to_live/set_up_a_live_smartpay_account) accounts.
+## Check your Worldpay account
 
-## Prerequisites
+Make sure your Worldpay account is both:
 
-Before you begin, make sure you have an admin account with both Worldpay and
-GOV.UK Pay. You need an account with the Worldpay Corporate Gateway. GOV.UK
-Pay does not support the Worldpay Business Gateway.
+- an admin account
+- using the Worldpay Corporate Gateway - not the Worldpay Business Gateway
 
-## Set up your merchant code
+## Switch to Production Mode for your merchant code
 
 Sign in to your [Worldpay
-account](https://secure.worldpay.com/sso/public/auth/login.html) and use
-__Select MerchantCode__ to find your merchant code.
+account](https://secure.worldpay.com/sso/public/auth/login.html) and select your merchant code.
 
-If __Switch to Production__ is at the bottom of the left-hand menu, toggle it on.
+If __Test Mode__ is at the bottom of the left-hand menu, switch to __Production Mode__. Worldpay uses the term 'production' for live accounts.
 
-## Set up your profile
+## Copy your details into GOV.UK Pay
 
-To set up your profile, you need to copy credentials from your Worldpay account
-into your GOV.UK Pay account.
+You must copy your details from your Worldpay account to your GOV.UK Pay live account.
 
-You must separately sign in to both your [Worldpay
-account](https://secure.worldpay.com/sso/public/auth/login.html) and your
-[GOV.UK Pay account](https://selfservice.payments.service.gov.uk/login). You
-should have both accounts open, in different web browser tabs.
+1. Sign in to the [__Your PSP - Worldpay__](https://selfservice.payments.service.gov.uk/your-psp) screen of the GOV.UK admin tool. Make sure you've selected your live account, not your test account.
 
-1. In your Worldpay browser tab, select __Profile__ in the left-hand menu. If you
-are prompted to choose a merchant code, select the appropriate one.
+2. In your Worldpay account, select __Account__, then the __Profile__ tab. Select your merchant code if you're asked for one.
 
-1. In your GOV.UK Pay browser tab, go to __My services__ to select the live
-service you want to set up. Select __Settings__, then __Your PSP - Worldpay__,
-then __Change__ under __Account credentials__.
+3. Copy the __Merchant Code__ value from the __Identification__ section of your Worldpay account into the __Merchant code__ field in your GOV.UK Pay account.
 
-1. In Worldpay, copy the Worldpay __Merchant Code__ from __Identification__ into
-the GOV.UK Pay __Merchant code__ field.
+4. Copy the __New Username__ value into the __Username__ field.
```

Comment:
Might need to clarify that _New Username_ is in Worldpay but _Username_ is in Pay.